### PR TITLE
Add VolumeMesh from local storage and Update module import for local tests

### PR DESCRIPTION
**Pull request by angranl-flex**
- Add all the modules that needs to be imported for localTests in `src/Flow360Core/Flow360/flow360/new_simulation_models.py` file.
- Update AssetBase to allow id=None for running tests locally.

### CODE REVIEW

File: flow360/component/simulation/web/asset_base.py [hunk 1]
```diff
@@ -31,21 +31,23 @@ class AssetBase(metaclass=ABCMeta):
     _entity_info: EntityInfoModel = None
 
     # pylint: disable=redefined-builtin
-    def __init__(self, id: str):
+    def __init__(self, id: str = None):
```
Comment (benflexcompute) [inline flow360/component/simulation/web/asset_base.py:34]: Let's not add default for id.

It does not bring any convenience but makes the interface more error-prone.

Also the default/most likely use case should not be when id is None.

```suggestion
    def __init__(self, id: Union[str, None]):
```

Comment (benflexcompute) [reply flow360/component/simulation/web/asset_base.py:34]: Add in docstring that when id is none the asset is meant to be operating in local mode.

File: flow360/component/volume_mesh.py [hunk 2]
```diff
@@ -1005,6 +1008,44 @@ def from_cloud(cls, id: str) -> VolumeMeshV2:
 
         return asset_obj
 
+    @classmethod
+    def from_local_storage(cls, case_id: str = None, local_storage_path="") -> VolumeMeshV2:
```
Comment (benflexcompute) [inline flow360/component/volume_mesh.py:1012]:
```suggestion
    def from_local_storage(cls, mesh_id: str = None, local_storage_path="") -> VolumeMeshV2:
```

Comment (angranl-flex) [reply flow360/component/volume_mesh.py]: Updated

File: flow360/component/simulation/web/asset_base.py (hunk 1 above)
Comment (angranl-flex) [reply flow360/component/simulation/web/asset_base.py]: Updated and also added the docstring

**Review by benflexcompute — APPROVED** [review]

**Pull request merged**